lib/Controller/DashboardController.php

```diff
@@ -39,11 +41,14 @@ public function __construct(string $AppName, IRequest $request){
 	}
 
 	/**
-	 * @NoAdminRequired
+	* @NoAdminRequired
 	* @NoCSRFRequired
+	*
+	* Render default template
 	*/
 	public function index() {
-		return new TemplateResponse('adminly_dashboard', 'dashboard');
+		Util::addScript(Application::APP_ID, 'adminly_dashboard-main');
```

Comment:
The appName is retrieved through dependency injection and automatically available as a variable through the parent constructor. So a better way to get the appName would be `$this->appName`;